src/test/java/tests/StoreTests.java

```diff
@@ -90,6 +93,44 @@ public void testStore() throws ExecutionException, InterruptedException {
         assertEquals(11, dummy2.getIssues().size());
     }
 
+    @Test(expected = ExecutionException.class)
+    public void testCorruptedJSON() throws InterruptedException, ExecutionException {
+        RepoStore.write("testrepo/testrepo", "abcde");
+
+        JSONStore jsonStore = new JSONStore();
+        jsonStore.loadRepository("testrepo/testrepo").get();
+
+        File f = new File("store/test/testrepo/testrepo");
+        f.delete();
+    }
+
+    @Test(expected = ExecutionException.class)
+    public void testNonExistedJSON() throws InterruptedException, ExecutionException {
```

Comment:
Lol pardon my English, I think being consistent should be good enough for these functions for now? I will probably check the words' definitions probably next time :p